addon/controllers/row-array.js

```diff
@@ -0,0 +1,25 @@
+import Ember from 'ember';
+
+export default Ember.ArrayController.extend({
+  itemController: null,
+  content: null,
+
+  rowContent: Ember.computed(function() {
+    return [];
```

Comment:
Elsewhere you use `Ember.A()`, why not here? (it makes a difference if prototype extensions are disabled)